_episodes/07-find.md

```diff
@@ -423,6 +423,8 @@ $ find .
 ~~~
 {: .output}
 
+> If you're seeing a file structure that's different than the one shown above, chances are you are not executing the find command in the appropiate folder (`data-shell/writing`). Reminder: you can use the `pwd` command to show what you're current path is. {: .output}
```

Comment:
> {: .output}
I can't see that this markup is needed? There's no code block it would be associated with.